packages/server/src/server.ts

```diff
@@ -337,6 +342,53 @@ app.get(
     },
 );
 
+app.get(
+    '/amp/choice_cards_data',
```

Comment:
why do we need a separate endpoint for this data? Can it not be included in the response from `/amp/epic`?

Reply:
@tomrf1 the `/amp/epic` endpoint returns data for the `amp-list`, and is not accessible from the `amp-state`. This endpoint is for retrieving data to populate the `amp-state`, which can then be bound to the choice card labels etc.

Reply:
haha yeah `amp` was never going to be able to handle something like that!

Reply:
That's interesting. So we initialise an amp-state from this request, and then the UI references/updates its values? (I probably need to see the dcr change to understand this!)
It makes me wonder if we can reduce this all to a single request (from amp-state?). It would be nice to avoid doubling AMP requests to SDC.

Reply:
If we could do that it would make things _much_ easier, but I don't think we can. I think we need to use an `amp-list` for the epic or we'll run into problems with its height.

Reply:
@tomrf1 I had an idea for combining these requests, which I was going to save for a refactor later on, but I think I'll give it a go now actually.

Reply:
It's disappointing given that the rendering all happens within the amp-list!
It would be good to get fastly to cache this response to save hitting SDC. I can help with this - it'll involve using the `X-GU-GeoIP-Country-Code` header for the cache key

Reply:
Nice - interested to hear how it goes! AMP is a mystery